### PR TITLE
Fix function declaration of TabController::get_key_array()

### PR DESCRIPTION
**Pull request by tsitle**
## Description

Declare the function TabController::get_key_array() as static function since
it is intended to be used as such and fix calls to this function that
assumed that it was a non-static function.

## Motivation and Context

Statically calling non-static functions is deprecated.

## How To Test This

1. In `php.ini` change the `error_reporting` setting from something like
```
error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
```
to
```
error_reporting = E_ALL & ~E_STRICT
```
2. Restart Apache (e.g. `service apache2 reload`) or PHP-FPM (e.g. `service php7.1-fpm restart`)
3. In SuiteCRM's Admin panel navigate to "Display Modules and Subpanels"
4. Check Apache error log for messages like
```
PHP Deprecated:  Non-static method TabController::get_key_array() should not be
called statically in
/var/www/html/suitecrm/include/SubPanel/SubPanelDefinitions.php on line 785
```
After applying this PR this message should not appear anymore.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by Mac-Rae** [comment]
Assessed :+1:

File: modules/MySettings/TabController.php [hunk 1]
```diff
@@ -85,13 +85,13 @@ public function get_system_tabs()
                         }
                     }
                     ACLController :: filterModuleList($tabs);
```
Comment (connorshea) [inline modules/MySettings/TabController.php]: Not really relevant to your PR, but I noticed this and went 😱 

```suggestion
                    ACLController::filterModuleList($tabs);
```

**Comment by tsitle** [comment]
Nice catch :-)

**Review by samus-aran — APPROVED** [review]

**Comment by samus-aran** [comment]
Accepting @connorshea review and PR merge :+1:

**Pull request merged**